src/braket/aws/aws_device.py

```diff
@@ -628,7 +631,8 @@ def get_devices(
                 warnings.warn(
                     f"{error_code}: Unable to search region '{region}' for devices."
                     " Please check your settings or try again later."
-                    f" Continuing without devices in '{region}'."
+                    f" Continuing without devices in '{region}'.",
+                    stacklevel=1,
```

Comment:
Do we want to enforce this argument being explicit? I don't have any examples of when I've wanted to use a non-default value here